src/main/ui/app/route/TagPage.jsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import {Component} from "react";
+import fetch from "isomorphic-fetch";
+import AddOnList from "../component/AddOnList";
+
+export default class TagPage extends Component {
```

Comment:
Looking at this, I wonder if we'd have less code duplication if we added this functionality to the existing search page...